tests/mpi/distribute_mg_dofs_periodicity_01.mpirun=28.output

```diff
@@ -0,0 +1,7 @@
+
+DEAL::Number of cells: 1024
```

Comment:
I think it is good to keep one 3d test where it is a bit more difficult to guess the level ghost layer.